tests/data_loader_config.py

```diff
@@ -0,0 +1,18 @@
+import os
```

Comment:
Note: This file is being added to make the tests run. Is this OK?

Reply:
Yes, I think it's fine for now